ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -338,8 +338,7 @@ objects:
             exit 1
         fi
 
-        /bin/openshift-install --dir=/tmp/artifacts/installer create cluster &
-        wait "$!"
+        /bin/openshift-install --dir=/tmp/artifacts/installer create cluster
```

Comment:
Why did you touch this?  Won't it break the `/tmp/exit`-setting trap we set at the beginning of this script?  Previous discussion in d862f6f12958 (#1957).

Reply:
ah, previous discussion [here][1].  I still think this change breaks the `trap`.

[1]: https://github.com/openshift/release/pull/2633/files#r249622809